frontend/javascripts/oxalis/view/right-border-tabs/dataset_info_tab_view.tsx

```diff
@@ -238,175 +239,99 @@ export class DatasetInfoTabView extends React.PureComponent<Props, State> {
     console.log(this.state.owningOrganizationDisplayName);
   }
 
-  getTracingStatistics() {
-    const statsMaybe = getStats(this.props.tracing);
-    return this.props.tracing.skeleton != null ? (
-      <div>
-        <p>Number of Trees: {statsMaybe.map((stats) => stats.treeCount).getOrElse(null)}</p>
-        <p>Number of Nodes: {statsMaybe.map((stats) => stats.nodeCount).getOrElse(null)}</p>
-        <p>Number of Edges: {statsMaybe.map((stats) => stats.edgeCount).getOrElse(null)}</p>
-        <p>
-          Number of Branch Points:{" "}
-          {statsMaybe.map((stats) => stats.branchPointCount).getOrElse(null)}
-        </p>
+  getAnnotationStatistics() {
+    if (this.props.isDatasetViewMode) return null;
+
+    const statsMaybe = getStats(this.props.annotation);
+    const treeCount = statsMaybe.map((stats) => stats.treeCount).getOrElse(null);
+    const nodeCount = statsMaybe.map((stats) => stats.nodeCount).getOrElse(null);
+    const edgeCount = statsMaybe.map((stats) => stats.edgeCount).getOrElse(null);
+    const branchpointCount = statsMaybe.map((stats) => stats.branchPointCount).getOrElse(null);
+    const segmentCount = 0; // TODO read value from actual stats data
+
+    return (
+      <div className="info-tab-block">
+        <p className="sidebar-label">Statistics</p>
+        <table className="annotation-stats-table">
+          <tbody>
+            <tr>
+              <td>
+                <img
+                  className="info-tab-icon"
+                  src="/assets/images/icon-skeletons.svg"
+                  alt="Skeletons"
+                />
+              </td>
+              <td>
+                <Tooltip
+                  placement="left"
+                  title={
+                    <>
+                      <p>Nodes: {nodeCount}</p>
+                      <p>Edges: {edgeCount}</p>
+                      <p>Branchpoints: {branchpointCount}</p>
+                    </>
+                  }
+                >
+                  <Typography.Text type="secondary">{treeCount} Skeletons</Typography.Text>
+                </Tooltip>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <img
+                  className="info-tab-icon"
+                  src="/assets/images/icon-segments.svg"
+                  alt="Segments"
+                />
+              </td>
+              <td>
+                <Typography.Text type="secondary">{segmentCount} Segments</Typography.Text>
```

Comment:
Maybe name these "registered segments", because this counts only the items in the segment list, right? or should we omit this information for now and make use of #7033 once it is merged? cc @fm3

Reply:
#7033 may not help with that a lot (it is supposed to provide statistics *per* segment). To scan through the entire index just to count the available segments does not sound great. Also, #7033 does not currently handle fallback segmentation layers at all. We may of course, in the future, add a total segment count as well. It is not something I’d await here, though.

So going with something like “registered segments” may be an option. However, I’m not super happy with that wording (since registering is so overloaded). I don’t really have a better idea though, “segments of interest” or “touched segments” don’t feel great either.

It may also be fair too remove this here entirely, since it does not really yield all that much information

Reply:
> getting the amount of keys in that dictionary-like structure in O(1) doesn't sound too far fetched

Sounds right, but I don’t know that fossildb can do that. We maybe it makes sense to save this info on top.


“Listed segments” is the best so far :)

Reply:
I added a tool tip.